hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/operations/CustomizedOpcodes.java

```diff
@@ -28,6 +28,7 @@ public enum CustomizedOpcodes {
     CHAINID(0x46),
     CREATE(0xf0),
     CREATE2(0xf5),
+    PREVRANDAO(0x44),
```

Comment:
Please leave a comment (maybe with link?) referencing [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399) - reason is even otherwise reliable sources like [evm.codes](https://www.evm.codes/?fork=shanghai) haven't been updated with `PREVRANDAO` yet.

Reply:
Added comment and link.